docs/sources/flow/reference/components/prometheus.operator.podmonitors.md

```diff
@@ -0,0 +1,193 @@
+---
```

Comment:
I don't think we have to put PodMonitor/PodMonitors or Pod/Pods in code backticks; that might make this render a little more nicely in the UI.